lib/Page.js

```diff
@@ -179,11 +179,11 @@ class Page extends EventEmitter {
    * @param {!Protocol.Log.entryAddedPayload} event
    */
   _onLogEntryAdded(event) {
```

Comment:
This is not the only entry point for console messages. There is also Runtime.consoleAPICalled. It has line number and url contained in the stack trace.

Reply:
can you please elaborate?
I'm looking into [`Runtime.consoleAPICalled`](https://chromedevtools.github.io/devtools-protocol/tot/Runtime#event-consoleAPICalled) and I don't see such details, nor a reference to [`ExceptionDetails`](https://chromedevtools.github.io/devtools-protocol/tot/Runtime#type-ExceptionDetails)

Reply:
It is contained in the callframes of the stacktrace: https://chromedevtools.github.io/devtools-protocol/tot/Runtime#type-CallFrame

Reply:
done